index.js

```diff
@@ -1413,6 +1413,27 @@ class Hydra extends EventEmitter {
   }
 
   /**
+   * @name _reconfigure
+   * @summary Sends a message to all present instances of a hydra service about the config changes
+   * @param {any} label in the format of 'my_service:1.0.0'
+   * @return {promise} Promise from sendBroadcastMessage
+   * @memberof Hydra
+   */
+  _reconfigure(label) {
+    let parts = label.split(':');
+
+    return this._sendBroadcastMessage(UMFMessage.createMessage({
+      to: `${parts[0]}:/`,
+      from: `${this.serviceName}:/`,
+      typ: 'reconfigure',
```

Comment:
You should be consistent in your usage of UMF short/long format. Here `from` and `body` are long format but `typ` is short format. You should use `frm` and `bdy`, or `type`.

See [here](https://github.com/cjus/umf/blob/master/umf.md#Short-form-syntax) for more info.